web/src/containers/admin/ManageAllUsersTable.js

```diff
@@ -56,9 +56,19 @@ const ManageAllUsersTable = ({
   }
 
   const AffiliationActions = ({ primaryAffiliation, affiliation }) => {
+    // Only show revoked for the active tab
+    let displayedActions;
```

Comment:
you could do this with a ternary

`const displayedActions = (tab === 'active') ? displayedActions = [actions[1]] : displayedActions = [...actions];`